src/profile/components/Settings/Membership/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { Pane } from 'common/components/Panes';
+import { Button } from '@dotkomonline/design-system';
+import MembersInfo from './Info';
+
+const Membership = () => {
+  return (
+    <Pane>
+      <MembersInfo></MembersInfo>
+      <Button
+        onClick={() => {
+          window.open('https://old.online.ntnu.no/profile/membership/');
+        }}
```

Comment:
I think you could wrap the `Button` in a `Link`-component, as in `<Link href="https://old.online.ntnu.no/profile/membership/"><Button>Medlemskap</Button></Link>` component instead of an `onClick`-function! (You might need to add `passRef={true}` to the `Link`, not completely sure tho)
`window.open` has a [security flaw](https://stackoverflow.com/questions/40593632/use-window-open-but-block-use-of-window-opener) which the `Link`-component handles for us already!
You can import `Link` from `next/link`.